### PR TITLE
[localisation] add Marathi language to the app

### PR DESCRIPTION
**Pull request by sanketgarade**
Hello

I have added Marathi strings for the OM app. Refer #2507 
Please check it and let me know if anything more needs to be done from my side.

Thanks 😃

### CODE REVIEW

**Review by vng — APPROVED** [review]
Great job! Please, rebase on latest master to fix conflicts.

**Comment by sanketgarade** [comment]
rebased and pushed. It seems there are some failed checks. Let me know if I can do anything to fix those.

**Comment by vng** [comment]
Check fails here: android/src/google/play/listings/mr-IN/title.txt
Title is expected to be <= 30 symbols.

**Comment by vng** [comment]
And I will update core categories code for "mr" later today or tomorrow.

**Comment by biodranik** [comment]
AppStore does not support all languages, unfortunately. Here is the list:

https://help.apple.com/app-store-connect/#/dev656087953

**Comment by vng** [comment]
So, make symlinks on EN or remove "iphone/metadata/mr-IN/" at all?

**Comment by sanketgarade** [comment]
> mr-IN for stores should be removed

Should I push a commit for this?

Also this is only for the iPhone store right?

**Comment by sanketgarade** [comment]
> Should I push a commit for this?
> 
> Also this is only for the iPhone store right?

@biodranik should I do this? (Sorry I didn't get any answer so I am just reconfirming).

**Comment by biodranik** [comment]
1. Linux tests should pass.
2. iOS checks should pass. Please move that (unsupported ATM) translation into metadata-unsupported folder.

**Comment by sanketgarade** [comment]
Some checks failed again. Anything that I can do to fix those?

**Comment by biodranik** [comment]
Please check failed tests output to save us a bit of time )

**Comment by sanketgarade** [comment]
I was able to fix some issues.

I have one question about the failed test case -
```
Running categories_test.cpp::CategoriesIndex_AllCategories
Number of nodes in the CategoriesIndex trie: 551668
FAILED
indexer_tests/categories_test.cpp:318 TEST(index.GetNumTrieNodes() < 450000) 551668 450000 
```

The number of nodes was 432729 in the base commit but after my additions it has become 551668.
Is this expected to happen? Or is something wrong?

If it is expected to happen, then is it ok to increase the pass criteria from 450000 to, say, 600000, which can make this PASS?

`TEST_LESS(index.GetNumTrieNodes(), 450000, ());`

**Comment by sanketgarade** [comment]
> I see different separators like "|" or "।". And I'm not sure that it will be interpreted correctly.

Oops, the small one `।` is wrong. I will fix it in sometime.

**Comment by vng** [comment]
Could you please make 2 commits: actual changes and generated mr strings? It will be much more convenient to revert changes if something will go wrong after merge.

**Comment by sanketgarade** [comment]
Done as requested.

**Comment by sanketgarade** [comment]
Now one check is failing.
But the code in latest commit is the exact same as previous one where all checks passed.
```
[marathi-app]organicmaps $ git diff 1b8f33e5f6 dd7097ccbb 
[marathi-app]organicmaps $ 
```
This is strange.

**Comment by biodranik** [comment]
@vng we need to fix this flaky test.

**Comment by vng** [comment]
- The check failed sometimes and its not concerning your changes.
- I mean to squash 20 commits into one, and make 2 commits only: all your changes as first commit and regenerated strings as second.

**Comment by biodranik** [comment]
Can you please fixup/squash commits?

**Comment by sanketgarade** [comment]
> Can you please fixup/squash commits?

Oh ok, it seems I misunderstood the previous comment. Anyway I'll do the squashing and create 2 commits later sometime today.

**Comment by sanketgarade** [comment]
Test failed - 
```
Running small_set_test.cpp::SmallMap_Benchmark1
FAILED
base_tests/small_set_test.cpp:140 TEST(BenchmarkTimeLessOrNear(t2, t1, 0.1)) 346 278
Test took 893 ms
```

Anything that I need to do? or can this too be ignored, like last time.

File: iphone/plist.txt [hunk 1]
```diff
@@ -53,6 +54,7 @@
     id = Penanda
     it = Segnalibri
     ja = ブックマーク
+    mr = खूणपत्रे
```
Comment (biodranik) [inline iphone/plist.txt:57]: Please make it in the alphabet order here and in other places.

Comment (sanketgarade) [reply iphone/plist.txt]: oops. I fixed it now in all places in this file and once again squashed into 2 commits and force pushed.

**Comment by sanketgarade** [comment]
Thanks for the merge and making this wonderful app!!

**Pull request merged**